cmd/mattermost/commands/command.go

```diff
@@ -34,13 +45,92 @@ var CommandListCmd = &cobra.Command{
 }
 
 func init() {
+	CommandCreateCmd.Flags().String("title", "", "Command Title")
+	CommandCreateCmd.Flags().String("description", "", "Command Description")
+	CommandCreateCmd.Flags().String("trigger-word", "", "Command Trigger Word (required)")
+	CommandCreateCmd.MarkFlagRequired("trigger-word")
+	CommandCreateCmd.Flags().String("url", "", "Command Callback URL (required)")
+	CommandCreateCmd.MarkFlagRequired("url")
+	CommandCreateCmd.Flags().String("creator", "", "Command Creator's Username (required)")
+	CommandCreateCmd.MarkFlagRequired("creator")
+	CommandCreateCmd.Flags().String("response-username", "", "Command Response Username")
+	CommandCreateCmd.Flags().String("icon", "", "Command Icon URL")
+	CommandCreateCmd.Flags().Bool("autocomplete", false, "Show Command in autocomplete list")
+	CommandCreateCmd.Flags().String("autocompleteDesc", "", "Short Command Description for autocomplete list")
+	CommandCreateCmd.Flags().String("autocompleteHint", "", "Command Arguments displayed as help in autocomplete list")
+	CommandCreateCmd.Flags().Bool("post", false, "Command Callback URL Method Type ")
```

Comment:
Could you change the help text for this flag to something like "Use POST method for Callback URL" as I think that makes more sense given it's a boolean flag.